relayer/middleware/source-tx.middleware.ts

```diff
@@ -36,15 +43,38 @@ const defaultOptsByEnv: { [k in Environment]: Partial<SourceTxOpts> } = {
   },
 };
 
+function ifVAAFinalized(vaa: ParsedVaaWithBytes) {
+  const { consistencyLevel, emitterChain } = vaa;
+  if (emitterChain === CHAIN_ID_SOLANA) {
+    return consistencyLevel === 32;
+  } else if (emitterChain === CHAIN_ID_BSC) {
+    return consistencyLevel > 15;
+  }
+  return consistencyLevel !== 200 && consistencyLevel !== 201;
+}
+
```

Comment:
added this in last commit

Reply:
I think I'd personally prefer to explicitly enumerate conditions chain by chain and then throw an error on unknown chains :thinking:
Or a safe default value? I don't know how this is used.

Reply:
I had a conversation with Evan to write this function, in almost all chains 200 and 201 are reserved for instant vaas, the exceptions are solana and bsc, that’s why it’s written this way.